framework/source/class/qx/ui/layout/VBox.js

```diff
@@ -315,12 +315,12 @@ qx.Class.define("qx.ui.layout.VBox",
 
 
       // Compute gaps
-      var spacing = this.getSpacing();
+      var spacing = this.getSpacing(), gaps;
```

Comment:
```suggestion
      var spacing = this.getSpacing();
      var gaps;
```

Seems odd to define it on the same line as spacing